NetworkRecurrentLayer.py

```diff
@@ -130,6 +130,142 @@ def step(self, i_t, x_t, z_t, y_p, c_p, *other_args):
     # return: next outputs (# unit.n_act, y_t, c_t, ...)
     return (y_t * i_output, c_t * i_h + c_p * (1 - i_h)) + tuple(other_outputs)
 
+class Leaky(Unit):
```

Comment:
Units need to be specified in all capital letters (the specification in the config file is case insensitive and will be capitalized to create the object). I.e. Leaky -> LEAKY, and maybe let's say LEAKYLSTM since the same idea could also be applied to GRUs for example.